.github/workflows/appimage-comment.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Comment on PR'
+
+on:
+  workflow_run:
+    workflows: ["Pull Request Update"]
+    types:
+      - completed
+
+jobs:
+  comment:
+    name: 'Post Comment on PR'
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.event == 'pull_request_target' && github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - name: Add AppImage Links
+        if: ${{ env.APPIMAGE }}
```

Comment:
Where are you getting the env from here? I think you deleted a couple of steps that were required. See the full set of steps in https://github.com/viamrobotics/rdk/blob/main/.github/workflows/code-coverage-comment.yml

Reply:
This still needs the download of the pr.env artifact first. See https://github.com/viamrobotics/rdk/blob/main/.github/workflows/code-coverage-comment.yml#L13-L18 

It's a poorly named step, but the key thing is that it's getting "pr.env" (a file) from the original workflow, and that contains the APPIMAGE and PR_NUMBER settings you're using here.

Reply:
Okay, I see what's missing in understanding possibly. I didn't realize your test workflow didn't have the markers for this already.

See this: https://github.com/viamrobotics/rdk/blob/main/.github/workflows/test.yml#L85-L106

It's the other side of the comment system. It's what's setting the APPIMAGE and PR_NUMBER variables, storing them in pr.env, and then uploading pr.env in the first place. You'll need that part too. Sorry I didn't realize you were missing the other half here, or I would've explained better/sooner.